lib/redlock/client.rb

```diff
@@ -124,7 +91,7 @@ class RedisInstance
         end
       eos
 
-      EXTEND_LOCK_SCRIPT = <<-eos
+      EXTEND_LIFE_SCRIPT = <<-eos
```

Comment:
@maltoe i hadn't looked into it yet, but I assumed that we could fix this in `LOCK_SCRIPT` and not have an additional script